packages/core/parcel-bundler/src/scope-hoisting/hoist.js

```diff
@@ -51,6 +51,8 @@ function hasSideEffects(asset, {sideEffects} = asset._package) {
 module.exports = {
   Program: {
     enter(path, asset) {
+      path.scope.crawl();
```

Comment:
I don't get why we need this, isn't babel traverse going to `.crawl` for us already?

Reply:
Here is an explanation: https://github.com/parcel-bundler/parcel/issues/1699#issuecomment-404248874

Reply:
I think babel doesn't recrawl the scope automatically, only when the scope is first initialized. That happens before lots of babel transforms run which can modify references, in this case the class transform. Then we get to our hoisting transform and we need to re-crawl to make sure we have updated references since babel traverse doesn't do that automatically.